config/cfme/manifest_5_11.json

```diff
@@ -24,58 +24,64 @@
       "vms": {
         "id": null,
         "name": null,
+        "description": null,
         "type": null,
-        "guid": null,
+        "ems_ref": null,
         "uid_ems": null,
         "archived": null,
-        "cpu_cores_per_socket": null,
+        "orphaned": null,
+        "retired": null,
         "cpu_total_cores": null,
+        "cpu_cores_per_socket": null,
         "disks_aligned": null,
-        "ems_ref": null,
         "has_rdm_disk": null,
-        "host": {
-          "ems_ref": null
-        },
```

Comment:
We need the `host.ems_ref` to connect VMs to hosts

Reply:
I removed this for the ManageIQ::Providers::OpenStack::CloudManager provider as they don't have hosts, but thinking about it they do have host aggregates which are sort of cluster-like. Should I add host aggregate for this provider type?

Reply:
Ah okay didn't see it was only removed for openstack, I'm good with this then.